content/aosc-os/installation/manual/arm-raspberry-pi.md

```diff
@@ -7,849 +8,4 @@ tags = ["sys-installation"]
-
-Panic messages are started with `kernel panic: not syncing` . Take a close look at this line, and find a match in follows.
-
-- `VFS: Unable to mount root fs on unknown-block(0,0)`
-
-  It seems that kernel can't find a root filesystem to boot.
-
-  **Check `root=` parameter in `cmdline.txt`**
-
-  If you installed AOSC OS in a SD card, the root filesystem path is like `/dev/mmcblk0pX` where X is a partition number. e.g. for this setup, it's `/dev/mmcblk0p2`.
-
-  Or if you installed AOSC OS in an USB drive, no matter what it is, the root filesystem path should be like `/dev/sdaX`.
-
-  > `UUID=` / `PARTUUID=` option are unavailable if you don't use an initramfs.
-
-  **Check if `rootwait` option exists in `cmdline.txt`**
-
-  Kernel can't find the root filesystem in USB/SD drives until such hardware begins to work. Add `rootwait` option will let kernel to wait until such drive comes up.
-
-- `Attempted to kill init!`
-
-  Your `init` program died during boot. Reinstall the OS may fix the problem.
-
-  Try mounting your media in your PC - if it fails, then your filesystem is corrupted.
-  
-  > Also check your `config.txt` and make sure it loads the correct 64-bit kernel. This means you should enable `arm_64bit=1` option and make sure it is uncommented,.
-
-
-## Running
-
-### Poor performance / Low Frequency
-
-If you are using the kernel obtained from Raspberry Pi's firmware repository, then you need to set CPU governor.
-
-> The default CPU governor configured in the kernel is `powersave`,  but other governors are available.
-
-To set a CPU governor, run:
-
-```
-sudo cpupower frequency-set -g <governor>
-```
-
-The `ondemand` and `conservative` is good enough for ARM processors. Or you can use `performance` if you have a good cooling system.
-
-### Long boot time due to network delay
-
-If your Pi boots fast but you should wait a few minutes to bring up the network, try to disable `NetworkManager-wait-online.service` and `systemd-networkd-wait-online.service` .
-
-```
-sudo systemctl disable NetworkManager-wait-online.service
-sudo systemctl disable systemd-networkd-wait-online.service
-```
-
-If you use Ethernet only, you can disable WiFi by `rfkill` as this would reduce the boot time:
-
-```
-sudo rfkill block wifi
-```
-
-
-# See also
-
-- [Raspberry Pi Documentation](https://www.raspberrypi.org/documentation/)
-- [Raspberry Pi GitHub](https://github.com/raspberrypi/)
-- [eLinux.org Raspberry Pi documentation](https://elinux.org/RPi_Hub)
-- [LibreELEC Source](https://github.com/LibreELEC/LibreELEC.tv)
-- [Arch Linux ARM Wiki](https://archlinuxarm.org/wiki/Raspberry_Pi)
+There is a [new article](https://wiki.aosc.io/aosc-os/devices/raspberrypi/4b/installation/) avaliable,so this article is no longer being maintained.
```

Comment:
1. Please add a white space after the comma.
2. Please add a new line at the end of the file.